src/main.cpp

```diff
@@ -26,6 +26,7 @@
 #include "crypto/Crypto.h"
 #include "gui/Application.h"
 #include "gui/MainWindow.h"
+#include "gui/csvImport/CsvImportWizard.h"
```

Comment:
You have to put this here because you forward-declared the CsvImporter class. Recommend NOT doing that in DatabaseWidget.cpp